libs/shared/selectors/src/selectors/sound-entries-selector/sound-entries-selector-btn.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { BsModalService } from 'ngx-bootstrap/modal';
+import { BaseSelectorBtnComponent } from '../base-selector/base-selector-btn.component';
+import { SoundEntriesSelectorModalComponent } from './sound-entries-selector-modal.component';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'keira-sound-entries-selector-btn',
+  templateUrl: '../base-selector/base-selector-btn.component.html',
+  styleUrls: ['../base-selector/base-selector-btn.component.scss'],
+  standalone: true,
+})
+export class SoundEntriesSelectorBtnComponent extends BaseSelectorBtnComponent {
+  /* istanbul ignore next */ // because of: https://github.com/gotwarlost/istanbul/issues/690
+  constructor(modalService: BsModalService) {
+    super(SoundEntriesSelectorModalComponent, modalService);
+  }
```

Comment:
needs to update because of https://github.com/azerothcore/Keira3/pull/2937 is now merged